webpack.config.js

```diff
@@ -107,7 +107,7 @@ const webpackConfig = ( mode ) => {
 			output: {
 				filename: '[name].js',
 				path: __dirname + '/dist/assets/js',
-				chunkFilename: '[name]-[chunkhash].js',
+				chunkFilename: '[name].js',
```

Comment:
Would this have any side effects regarding browser caching? We used to have tons of issues with blank screens because of partially old resources being loaded alongside new ones, and I'm a bit worried that removing the chunk hash (specifically necessary for `googlesitekit-vendor.js` would bring back some of these problems. What do you think @tofumatt?

Reply:
Would be great to keep the `chunkhash` on all chunk files, except `googlesitekit-vendor.js` which will receive the WordPress `?ver=...` parameter for cache busting.

Reply:
Good catch. I've pushed a change that only omits the hash for `googlesitekit-vendor.js`. 👍